admin/listPurchases/listView.html

```diff
@@ -404,14 +408,14 @@
     `;
 }
 
-function loadSomeOrders(){
-    requestOrdersWithQuery(createQueryString(), (error, orders) => {
+function getOrders(callback){
+    requestOrdersWithQuery(createQueryString(), (error, response) => {
         if(error) {
             return console.error("Failed to load entries", error);
         }
-        entries = orders.orders;
-        amountOfEntries = orders.amountOfOrders;
-        updateContent();
+        entries = response.orders;
+        amountOfEntries = response.amountOfOrders;
+        callback();
```

Comment:
Do you need `entries` top-level now? Or could you simply pass them as a result to the callback, which is where they are used?